examples/rpc/rpc_from_vcn_module/main.tf

```
@@ -0,0 +1,143 @@
+# Copyright (c) 2019, 2021, Oracle Corporation and/or affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/
+
+# Version requirements
+
+terraform {
+  required_providers {
+    oci = {
+      source  = "hashicorp/oci"
+      version = ">=4.41.0"
+    }
+  }
+  required_version = ">= 1.0.0"
+}
+
+# Resources
+
+module "vcn_local" {
+  # this module use the generic vcn module and configure it to act as local vcn
+  source  = "../../../"
```

Better to use the terraform public registry path in examples so the example is ready to run without requirement to clone the git repo first

```suggestion
  source  = "oracle-terraform-modules/vcn/oci"
```